app/controllers/v1/sessions_controller.rb

```diff
@@ -133,12 +134,12 @@ def user_login(saml_response)
         @current_user, @session_object = user_session_form.persist
         set_cookies
         after_login_actions
-        redirect_to url_service.login_redirect_url(saml_uuid: user_session_form.saml_uuid)
+        redirect_to url_service(user_session_form.saml_uuid).login_redirect_url
         if location.start_with?(url_service.base_redirect_url)
           # only record success stats if the user is being redirect to the site
           # some users will need to be up-leveled and this will be redirected
           # back to the identity provider
-          login_stats(:success, saml_response)
+          login_stats(:success, saml_response, user_session_form)
```

Comment:
If we're only using the saml_uuid in login_stats, could consider only passing that as the parameter :shrug:

Reply:
`login_stats` uses the `user_session_form` in the failure block